projects/angular/src/data/datagrid/render/main-renderer.ts

```diff
@@ -287,4 +289,14 @@ export class DatagridMainRenderer implements AfterContentInit, AfterViewInit, Af
       this.columnsSizesStable = true;
     }
   }
+
+  private setVisibility() {
+    if (this.el.nativeElement.offsetParent) {
+      this.isVisible = true;
+      this.datagrid.datagrid.nativeElement.style.visibility = 'visible';
```

Comment:
Setting it to empty string is enough. Test it just in case.
```suggestion
      this.datagrid.datagrid.nativeElement.style.visibility = '';
```

Reply:
I think what Daniel has is better. Setting it to `'visible'` has a obvious effect. Setting it to an empty string might do the same thing, but it not as readable.